extensions/http/src/main/java/org/mule/extension/http/internal/request/RequestConnectionParams.java

```diff
@@ -116,6 +128,10 @@ public Integer getConnectionIdleTimeout() {
     return connectionIdleTimeout;
   }
 
+  public Integer getResponseBufferSize() {
```

Comment:
I'll check, I think it's just following the other fields but we may be able to change it.